src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -213,6 +210,22 @@ protected void onResume()
         mPrefs.registerOnSharedPreferenceChangeListener( this );
         mMogaController.onResume();
         refreshViews();
+        
+        if( ACTION_RESUME.equals( action ) )
```

Comment:
I think I was going to wait to move it since it needs mGamePrefs, mRomHeader, mRomDatabase, etc., and has a nasty bit involving a player map dialog that I've never used before.